lib/vrl/stdlib/src/parse_datadog_grok.rs

```diff
@@ -0,0 +1,325 @@
+use datadog_grok::parse_grok_rules::Error as GrokError;
+use datadog_grok::parse_grok_rules::GrokRule;
+use datadog_grok::{parse_grok, parse_grok_rules};
+use std::fmt;
+use vrl::{
+    diagnostic::{Label, Span},
+    prelude::*,
+};
+
+#[derive(Debug)]
+pub enum Error {
+    GrokParsingError(GrokError),
+}
+
+impl fmt::Display for Error {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            Error::GrokParsingError(err) => write!(f, "{}", err.to_string()),
+        }
+    }
+}
+
+impl std::error::Error for Error {}
+
+impl DiagnosticError for Error {
+    fn code(&self) -> usize {
+        109
```

Comment:
Has error 109 been documented anywhere? I noticed https://errors.vrl.dev jumps from 108 to 110.

Reply:
Hm, nice catch, I copied it from 'parse_grok'. @StephenWakely(as the author of the "standard" function) - do you think we need to document this error message?